project/src/main/java/com/google/sps/servlets/PollingLocationsServlet.java

```diff
@@ -0,0 +1,86 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import java.net.URLEncoder;
+
+@WebServlet("/pollingLocation")
+public class PollingLocationsServlet extends HttpServlet {
+  private final static String VOTER_INFO_QUERY_URL =
+    String.format("https://www.googleapis.com/civicinfo/v2/voterinfo?key=%s",
+                  Config.CIVIC_INFO_API_KEY);
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userAddress = request.getParameter("address");
+    String encodedAddress = URLEncoder.encode(userAddress);
+    String queryUrl = String.format("%s&address=%s", VOTER_INFO_QUERY_URL, encodedAddress);
+    JsonObject voterInfoJson = queryCivicInformation(queryUrl);
```

Comment:
Consider catching and handling the IOException potentially thrown by `queryCivicInformation`. You can return an empty JSON back